src/index.js

```diff
@@ -1,12 +1,13 @@
+import './index.css';
+import navigation from './modules/navigation.js';
+import books from './modules/books.js';
+import displayBook from './modules/displayBook.js';
 
+const { DateTime } = require('luxon');
 
-function component() {
-  const element = document.createElement('div');
+const timeContent = document.getElementById('time');
+timeContent.textContent = DateTime.now().toISO();
 
-  // Lodash, currently included via a script, is required for this line to work
-  element.innerHTML = _.join(['Hello', 'webpack'], ' ');
-
-  return element;
-}
-
-document.body.appendChild(component());
\ No newline at end of file
+displayBook();
+navigation();
```

Comment:
- [ ] navigation is not a function, calling it here results in a compilation error. Please take another look at the codebase and figure out what you intended to do here.